2021-07-28 02:41:25,423 WARN  [org.keycloak.events] (default task-391) type=LOGIN_ERROR, realmId=Procurite, clientId=procurite-web, userId=null, ipAddress=127.0.0.1, error=user_not_found, auth_method=openid-connect, auth_type=code, redirect_uri=https://qa.procurite.id/purchaseorder, code_id=0bbfc372-749f-4a90-b594-f9bcd4d1dc8b, username=sennay01@yopmail.com, authSessionParentId=0bbfc372-749f-4a90-b594-f9bcd4d1dc8b, authSessionTabId=BVGO7qATnOs
2021-07-28 06:10:09,362 WARN  [org.jboss.jca.core.connectionmanager.pool.strategy.OnePool] (Timer-2) IJ000621: Destroying connection that could not be validated: org.jboss.jca.core.connectionmanager.listener.TxConnectionListener@79acb8ba[state=NORMAL managed connection=org.jboss.jca.adapters.jdbc.local.LocalManagedConnection@343185fc connection handles=0 lastReturned=1627451710834 lastValidated=1627015395340 lastCheckedOut=1627451710832 trackByTx=false pool=org.jboss.jca.core.connectionmanager.pool.strategy.OnePool@4033a96a mcp=SemaphoreConcurrentLinkedQueueManagedConnectionPool@6d9c9bfe[pool=KeycloakDS] xaResource=LocalXAResourceImpl@20f060d4[connectionListener=79acb8ba connectionManager=77e9fb08 warned=false currentXid=null productName=MySQL productVersion=8.0.25-0ubuntu0.20.04.1 jndiName=java:jboss/datasources/KeycloakDS] txSync=null]
